.travis.yml

```diff
@@ -66,10 +66,10 @@ jobs:
     env:
     - NODE_ENV=production
     - PORT=12345
-    - STREAMR_API_URL=https://ee-staging.streamr.com/api/v1
-    - STREAMR_WS_URL=wss://ee-staging.streamr.com/api/v1/ws
-    - STREAMR_URL=https://ee-staging.streamr.com
-    - PLATFORM_ORIGIN_URL=https://marketplace-staging.streamr.com
+    - STREAMR_API_URL=https://staging.streamr.com/api/v1
+    - STREAMR_WS_URL=wss://staging.streamr.com/api/v1/ws
+    - STREAMR_URL=https://staging.streamr.com
+    - PLATFORM_ORIGIN_URL=https://staging.streamr.com/marketplace
```

Comment:
PLATFORM_ORIGIN_URL should be consistently be the root URL of the domain w/ no trailing slash. 

PLATFORM_ORIGIN_URL=https://staging.streamr.com ~/marketplace~

Reply:
I think we can configure the /path like: `PLATFORM_PUBLIC_PATH=/marketplace`